packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -151,11 +151,51 @@ abstract class GestureRecognizer extends GestureArenaMember with DiagnosticableT
 /// simultaneous touches to each result in a separate tap.
 abstract class OneSequenceGestureRecognizer extends GestureRecognizer {
   /// Initialize the object.
-  OneSequenceGestureRecognizer({ Object debugOwner }) : super(debugOwner: debugOwner);
+  ///
+  /// {@template flutter.gestures.oneSequenceGestureRecognizer.kind}
+  /// It's possible to limit this recognizer to a specific [PointerDeviceKind]
+  /// by providing the optional [kind] argument. If [kind] is null,
+  /// the recognizer will accept pointer events from all device kinds.
```

Comment:
Describe what can be done if you want 2 out of the 5 or some such.

Reply:
Currently that's not supported. I didn't want to do it until I've seen a use case (YAGNI).